### DISCUSSION TITLE
stop assigning everyone on every issue

### DISCUSSION THREAD

**Discussion by nmsderp** [cytrinc/bark-coding#101] [Admin General]
it floods my notifications on gh, please stop, only assign people who asked to be assigned or are well known for making that certain part

**Comment by raynepaws** [comment]
i noticed this too, i'm being assigned to issues that say **in the description** "only mariocraft can do this"

**Reply by nmsderp** [reply]
yeah its annoying

not to be rude but @Rubiidev-18 does it like every time

**Reply by nmsderp** [reply]
![image](https://github.com/mariocraft987/bark.github.io/assets/130254323/5806aa27-4e2e-41a6-a16b-7bbe6758e7e2)

**Reply by mariocraft987** [reply]
so true

**Reply by dumorando** [reply]
yeah it used to happen to me it was rly annoying
